src/Stub.php

```diff
@@ -416,7 +416,9 @@ public static function constructEmptyExcept(
     private static function generateMock()
     {
         $args = func_get_args();
-        if (version_compare(PHPUnitVersion::series(), '10.4', '>=') && !is_bool($args[1])) {
+        if (version_compare(PHPUnitVersion::series(), '11', '>=') && !is_bool($args[1])) {
+            array_splice($args, 1, 0, [true, true]);
```

Comment:
Do you have a reference to the updated mock call in PHPUnit that requires this change?

Reply:
Sure, it's the parameter `$markAsMockObject` in https://github.com/sebastianbergmann/phpunit/blob/ece3536c22fc5113906a42e7e82de00baaef36d0/src/Framework/MockObject/Generator/Generator.php#L111 which was added in https://github.com/sebastianbergmann/phpunit/commit/db9ae302fe1ad89451ecfacc850e88ab7c6df5a3